YIF.Core.Data/Entities/Lector.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+using YIF.Core.Data.Entities.IdentityEntities;
+
+namespace YIF.Core.Data.Entities
+{
+    public class Lector : BaseEntity
+    {
+        public string InstitutionOfEducationId { get; set; }
+
+        [ForeignKey("InstitutionOfEducationId")]
+        public InstitutionOfEducation InstitutionOfEducation { get; set; }
+
+        [ForeignKey("UserId")]
+        public string UserId { get; set; }
+        public DbUser User { get; set; }
+    }
+}
```

Comment:
Why do you haven't added migration for changing the table name?

Reply:
https://forums.asp.net/t/1961622.aspx?Migrations+renaming+table+without+drop

Reply:
As for me, the main point with it is that we track changes of our db by migrations, so we can see what changes what added, so when we rename table is also important, and when we will have hiden changed lines in Snapshot, nobody will look through all big Snapshot file, because going through migrations is more readable.